netci.groovy

```diff
@@ -58,12 +58,13 @@ def addPushJob(String project, String branch, String os, String configuration)
   };
 };
 
+// Tarball builds that are not enforced to be offline
 [true, false].each { isPR ->
   ["RHEL7.2"].each { os->
     ["Release", "Debug"].each { configuration ->
 
-      def shortJobName = "${os}_Offline_${configuration}";
-      def contextString = "${os} Offline ${configuration}";
+      def shortJobName = "${os}_Tarball_${configuration}";
+      def contextString = "${os} Tarball ${configuration}";
```

Comment:
Do we need to keep the tarball build that isn't enforced to be offline?

Reply:
Hmm, I guess we technically don't but it might be good to have around to compare if we get any unusual errors.  I'll leave it in for now and log an issue for removing it after we have the offline build working all the way through.

Reply:
Sounds good.

Reply:
Issue is https://github.com/dotnet/source-build/issues/373.